mlflow/tracing/constant.py

```diff
@@ -7,6 +7,7 @@ class TraceMetadataKey:
 
 class TraceTagKey:
     TRACE_NAME = "mlflow.traceName"
+    EVAL_REQUEST_ID = "eval.requestId"
```

Comment:
A random question. Should this be prefixed with `mlflow` or not?

Reply:
any value that doesn't start with `mlflow.`?

Reply:
Yes, it's just a tag as a reference so they can link it to the correct question